patch-package
--- a/node_modules/fengari/src/lvm.js
+++ b/node_modules/fengari/src/lvm.js
@@ -651,11 +651,12 @@ const luaV_execute = function(L) {
                 }
 
                 let h = L.stack[ra].value;
-                let last = ((c - 1) * LFIELDS_PER_FLUSH) + n;
+                let last = ((c - 1) * LFIELDS_PER_FLUSH) + 1;
 
-                for (; n > 0; n--) {
-                    ltable.luaH_setint(h, last--, L.stack[ra + n]);
+                for (let i =0; i < n; i++) {
+                    ltable.luaH_setint(h, last++, L.stack[ra +  i+1]);
                 }
+
                 ldo.adjust_top(L, ci.top);  /* correct top (in case of previous open call) */
                 break;
             }
